homeworks/sasha.demidenko_SashaDemi/GamesProject/index.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Games</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="block-text">
+            <h1 class="game-title">About me</h1>
+            <div class="text-content">
+                <p>
+                    Hi! My name is Sasha, and I'm a Junior Frontend Developer.
+                    I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+                </p>
+                <p>
+                    This page was developed during the course
+                    <a href="https://www.mastersacademy.education/frontend-for-beginners-it">
+                        Frontend for beginners' from Masters Academy in 2023.</a>
+                </p>
+                <p>
+                    This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to
+                    create my own small project for the portfolio.
+                </p>
+                <p>
+                    You can contact me via
+                    <a href="https://www.instagram.com/sasha.dmk/">Instagram</a>
```

Comment:
please add classes  for your links tags and rewrite style for it. It will be more convenient in the future as you can have other links with different styles

Reply:
thanks, thanks, fixed it